app/models/volunteer.rb

```diff
@@ -28,9 +29,10 @@ class Volunteer < ApplicationRecord
 
   STATES = STATES_FOR_REVIEWED.dup.unshift(REGISTERED).freeze
 
+  enum acceptance: [:undecided, :accepted, :rejected, :resigned]
+
   belongs_to :user, -> { with_deleted }, optional: true
-  belongs_to :registrar, optional: true,
-    class_name: 'User', foreign_key: 'registrar_id'
+  belongs_to :registrar, optional: true, class_name: 'User', foreign_key: 'registrar_id'
```

Comment:
`foreign_key` is optional here, default is based on the association name (just FYI, I see you were just reformatting)

Reply:
I just uncluttered the line, didn't change anything